### PR TITLE
Up 171

### PR DESCRIPTION
**Pull request by ConnorHibbs**
Implemented the pagination code in the inventory page. Inventory is currently locked to 25 items but can be made more variable in the future if that is something we want. There are some other fixes to miscellaneous files because as I was trying to find where things in the inventory were defined I went through a couple higher up template files and fixed what I could along the way. The functional changes should all be within inventory though.

### CODE REVIEW

File: packages/app/pages/Inventory/inventory.scss [hunk 1]
```diff
@@ -111,3 +129,13 @@
          }
      }
 }
+//
```
Comment (fontainesw) [inline packages/app/pages/Inventory/inventory.scss]: Does this need to be here?

File: packages/app/pages/Inventory/inventory.tsx [hunk 2]
```diff
@@ -75,34 +76,94 @@ class InventoryCard extends React.Component {
...
+
+    return (
+      <aside {...currentTheme(1, "sidebar")} >
+        <div {...currentTheme(2, "sidebar_menu")} >
+          <h4 {...currentTheme(4, "menu_title")} >
+            Inventory List <span>{this.agent.ownedEconomicResources.length}</span>
+          </h4>
+        </div>
+
+        <PageNumbers numPages={numPages} current={page} updatePage={this.updatePage} theme={this.theme} />
```
Comment (fontainesw) [inline packages/app/pages/Inventory/inventory.tsx:161]: The definition for PageNumbers should appear before it is called

**Review by fontainesw — COMMENTED** [review]
Looks pretty good. I just had 2 things for you to look at

**Review by Lou3797 — APPROVED** [review]

**Review by Ryan-Guinn — APPROVED** [review]

File: packages/app/pages/Inventory/inventory.scss (hunk 1 above)
Comment (ConnorHibbs) [reply packages/app/pages/Inventory/inventory.scss]: Nope! I removed it and the commented out CSS underneath. That was left over from an earlier styling attempt.

File: packages/app/pages/Inventory/inventory.tsx (hunk 2 above)
Comment (ConnorHibbs) [reply packages/app/pages/Inventory/inventory.tsx:161]: Did not know that was a thing for JS. It's been moved up.

**Review by fontainesw — APPROVED** [review]
Looks good!

**Pull request merged**